packages/dcos-integration-test/extra/test_endpoints.py

```diff
@@ -279,7 +281,7 @@ def _validate_dcos_overlay(overlay_name, agent_overlay, master_agent_overlay):
     assert nested_match(expected, agent_overlay)
 
 
-def _validate_overlay_subnet(agent_subnet, overlay_subnet, prefixlen):
+def _validate_overlay_subnet(agent_subnet: str, overlay_subnet: object, prefixlen: Any) -> None:
```

Comment:
What's the difference between `Any` and `object` in mypy?

Reply:
Any also can be used, object is more specific, will use Any here because of your recommendation